### PR TITLE
Motor reorder dialog fix: importing serial backend

### PR DESCRIPTION
**Pull request by limonspb**
Dialog was crashing at "SAVE" click.

### CODE REVIEW

**Comment by haslinghuis** [comment]
Already fixed in https://github.com/betaflight/betaflight-configurator/pull/3214 :)

**Comment by limonspb** [comment]
> Already fixed in #3214 :)

that one still a draft, but people need to test :)

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by blckmn — APPROVED** [review]

**Pull request merged**
